perftrackerlib/client.py

```diff
@@ -462,7 +462,7 @@ def __init__(self, name, version=None, node_type=None, ip=None, hostname=None, p
             if not self.hostname:
                 self.hostname = self._shell.os_info.hostname
             if not self.ram_mb:
-                self.ram_mb = int(round(self._shell.hw_info.ram_kb / 1024, 0))
+                self.ram_mb = int(round(int(self._shell.hw_info.ram_kb) / 1024, 0))
```

Comment:
ram_kb must be an int itself, not just ram_mb

Reply:
You can see an exception I've got on screenshot below.
![screenshot 2018-11-12 at 20 40 41](https://user-images.githubusercontent.com/13187194/48365172-3d81f500-e6bb-11e8-965a-a516c4a81d44.png)